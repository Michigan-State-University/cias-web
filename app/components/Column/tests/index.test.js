import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Column } from '../index';

describe('<Column />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer.create(<Column />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
