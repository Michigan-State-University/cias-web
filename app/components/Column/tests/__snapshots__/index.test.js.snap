// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> should match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
}

@media only screen and (min-width:768px) {

}

@media only screen and (min-width:960px) {

}

@media only screen and (min-width:1170px) {

}

@media only screen and (min-width:1360px) {

}

<div>
  <div
    class="c0"
  />
</div>
`;
